projects/ngneat/hotkeys/src/lib/hotkeys.service.ts

```diff
@@ -115,11 +128,17 @@ export class HotkeysService {
   registerHelpModal(openHelpModalFn: () => void, helpShortcut: string = '') {
     this.addShortcut({ keys: helpShortcut || 'shift.?', showInHelpMenu: false, preventDefault: false }).subscribe(e => {
       const skipMenu =
-        /^(input|textarea)$/i.test(document.activeElement.nodeName) || (e.target as HTMLElement).isContentEditable;
+        /^(input|textarea|select)$/i.test(document.activeElement.nodeName) ||
```

Comment:
Why not reuse this logic?

Reply:
document.activeElement.nodeName doesn't work when testing. I guess you can type text in an input programatically without being active.

Reply:
We can mock it on test. Updating the value programatically will not trigger the callback. Check it.

Reply:
I'm using spectator's focus, but doesn't seem to change the document active element.

```
    spectator.focus(spectator.element);
    spectator.dispatchKeyboardEvent(spectator.element, 'keydown', 'a', spectator.element);
```

Reply:
Yes, just do `document.activeElement.nodeName = value`